test/parallel/test-fs-mkdtemp.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+
+const common = require('../common');
+const assert = require('assert');
+const fs = require('fs');
+const path = require('path');
+const Buffer = require('buffer').Buffer;
+
+common.refreshTmpDir();
+
+const tmpFolder = fs.mkdtempSync(path.join(common.tmpDir, 'foo.'));
+
+assert(path.basename(tmpFolder).length === 'foo.XXXXXX'.length);
+assert(common.fileExists(tmpFolder));
+
+const utf8 = fs.mkdtempSync(path.join(common.tmpDir, '\u0222abc.XXXXXX'));
+assert(Buffer.byteLength(path.basename(utf8)),
+       Buffer.byteLength('\u0222abc.XXXXXX'));
```

Comment:
Did you mean to use `assert.equal` here? Otherwise this just makes sure that byte length if utf8 is non-zero.